tools/generate_schema_wrapper.py

```diff
@@ -30,7 +30,7 @@
 
 # Map of version name to github branch name.
 SCHEMA_VERSION = {
-    "vega-lite": {"v5": "v5.14.1"},
+    "vega-lite": {"v5": "v5.15.0"},
```

Comment:
Just a heads-up that I'm simplifying this part of the code in #3207 and will merge it soon -> You will have a merge conflict here which should be simple to resolve. You can then just set `SCHEMA_VERSION = "v5.15.0"` as we no longer need the dictionary.